github/helpers/githubSDK.ts

```diff
@@ -527,26 +527,30 @@ export async function getUserAssignedIssues(
 export async function getIssueData(
     repoInfo: String,
     issueNumber: String,
-    access_token: String,
+    access_token: String | null,
     http: IHttp
 ): Promise<IGitHubIssue> {
     try {
-        let result = await http.get(
-            `https://api.github.com/repos/${repoInfo}/issues/${issueNumber}`,
-            {
-                headers: {
-                    "Content-Type": "application/json",
-                },
+        if(access_token){
+            var response = await getRequest(http, access_token, BaseRepoApiHost + repoInfo + '/issues/' + issueNumber);
+        }else{
+            const result = await http.get(
+                `https://api.github.com/repos/${repoInfo}/issues/${issueNumber}`,
+                {
```

Comment:
We can keep the variable names uniform (`result`/`response`) and clean this code.